google-cloud-vision/lib/google/cloud/vision/v1/doc/overview.rb

```diff
@@ -36,15 +36,15 @@ module Cloud
     #
     # ### Installation
     # ```
-    # $ gem install google-cloud-vision
+    # $ gem install google-cloud
```

Comment:
Argh... What's going on?? This needs to be reverted (and probably another bug filed against gapic-generator).